notifier/main.ts

```diff
@@ -75,7 +75,10 @@ app.use(async (ctx, next) => {
   }
 });
 
-router.get(`/${bot.token}`, webhookCallback(bot, "oak"));
+router.get(
+  `/${bot.token.replaceAll(":", "\:")}`,
```

Comment:
Why not this?
```suggestion
  `/${bot.token.replaceAll(":", "\\:")}`,
```

Reply:
Your previous review was really improper this time. But thanks for correcting it!

Reply:
I edited immediately after commenting, but you were faster in applying it

Reply:
Thanks, excuse my idiotism today!